examples/remix-auth-email-link/README.md

```diff
@@ -0,0 +1,18 @@
+# Remix Auth Email Link
+
+Passwordless authentication by sending a magic link to your email. This example is using [remix-auth](https://github.com/sergiodxa/remix-auth) and [remix-auth-email-link](https://github.com/pbteja1998/remix-auth-email-link) packages.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth-email-link)
```

Comment:
Several people are making examples of strategies with remix-auth and we decided we want to group them in a `examples/remix-auth` directory. Could you move this example into a sub directory (`examples/remix-auth/email-link`)?

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/email-link)
```